build.rs

```diff
@@ -13,6 +13,10 @@ fn apply_template(template: &Path) -> String {
         PathBuf::from(env::var_os("OUT_DIR").expect("OUT_DIR environment variable not defined"));
     fs::create_dir_all(&outdir).expect("unable to create out dir");
 
+    let script_base_dir = PathBuf::from(env!(
+        "SCRIPTDIR",
+        "SCRIPTDIR environment variable not defined"
+    ));
```

Comment:
```suggestion
    let script_base_dir = PathBuf::from(env!(
        "SCRIPTDIR",
        "SCRIPTDIR environment variable not defined"
    ));
    println!("cargo:rerun-if-env-changed=SCRIPTDIR");
    println!("cargo:rerun-if-changed=.cargo/config.toml");
```

Otherwise the binary won't be rebuilt if you change the env variable.

Reply:
Ok, done.